### ISSUE TITLE
Tối ưu siêu tham số huấn luyện

### ISSUE THREAD

**Issue by yenq89** [yenq89/SE2025-14.2#30]
Các tham số cần tối ưu là:
- train_batch_size 
- num_train_epochs 
- learning_rate 
- lr_scheduler 

Bộ tham số cần test, loss và tập ảnh validation được lưu lại làm cơ sở đánh giá trong folder wandb.
